catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -473,20 +500,13 @@ public Serializable getDocValue(String solrFieldName, Object docValue) {
       return Short.parseShort(docValue.toString());
     } else if (AttributeFormat.OBJECT.equals(format)) {
 
-      ByteArrayInputStream bais = null;
-      ObjectInputStream in = null;
-      try {
-        bais = new ByteArrayInputStream((byte[]) docValue);
-        in = new ObjectInputStream(bais);
+      try (ByteArrayInputStream bais = new ByteArrayInputStream((byte[]) docValue);
+          ObjectInputStream in = new ObjectInputStream(bais)) {
         return (Serializable) in.readObject();
       } catch (IOException e) {
         LOGGER.info("IO exception loading input document", e);
       } catch (ClassNotFoundException e) {
         LOGGER.info("Could not create object to return.", e);
```

Comment:
If you want to make some more improvements in here we could fix these log levels. maybe `debug`?